change/@microsoft-fast-foundation-4e5a154f-162b-48d4-84fb-df1ddc03e213.json

```
@@ -0,0 +1,7 @@
+{
+  "type": "patch",
+  "comment": "fix: selected option is not visible when dropdown is opened in fast-select",
```

Same note as above, leaving the suggestion only
```suggestion
  "comment": "selected option is not visible when dropdown is opened in fast-select",
```